blog/2021-06-14-stryker4s-sbt-performance.mdx

```diff
@@ -0,0 +1,151 @@
+---
+slug: stryker4s-sbt-speedup
+title: 'Improving Stryker4s performance: from 40 minutes to 40 seconds'
+author: Hugo van Rijswijk
+author_title: Stryker4s Team
+author_url: https://github.com/hugo-vrijswijk
+author_image_url: https://avatars.githubusercontent.com/u/10114577
+tags: [stryker4s]
+---
+
+Stryker4s is your friendly mutation testing framework for Scala. Compared to earlier versions, Stryker4s `0.12` achieves a 100x speed increase 🤯. Are you curious to know how? Keep on reading.
+
+In this blog we'll talk about the performance improvements made in the last couple Stryker4s versions and dive into how they are made possible.
+
+<!--truncate-->
+
+If you're new to mutation testing, it's a way to measure your tests' effectiveness. A mutation testing framework will make small changes, called _mutants_, one by one in your source code. Then it will run your tests to see if one of them fails. If so, you just "killed" that mutant; if not, it "survived". If too many mutants survive, you probably want to improve your tests. The mutation testing report will give you insides into the test cases you may have missed. If this all sounds complicated, please take a look at the recent talk ["Who is testing your tests?"](https://www.youtube.com/watch?v=Vq9eqZzblfg) from _Scala Love in the City_ or check out the rest of the [website](/).
+
+Imagine you're on your way to a sunny holiday destination. You've checked in your luggage, gone through customs, and boarded the plane. Then, the plane moves forward a couple of meters, and you're thrown out and sent back to customs before going through the whole thing again. It'll take years before you get to sip on your umbrella cocktail 🍹! This is exactly how Stryker4s has been running your code… until now.
+
+## 📃 In short
+
+A lot has changed in recent versions of Stryker4s. Stryker4s now runs your tests in an entirely different way, bringing big performance improvements. As an example, running Stryker4s on the Stryker4s codebase used to take about 40 minutes on `0.9.1`. On`0.12.1`, this has been reduced to a fast 40 seconds. A short list of some new features:
+
+- ✨ New testrunner boasting improved performance
+- 🔀 Concurrency for even faster performance
```

Comment:
Adjectives like "even" (faster), "a lot", "very", etc often have the opposite effect on the reader. They might think: "wait a second, is it _really_ faster? Wasn't it fast before?". 

Suggestion: "Concurrency to improve speed by using all available cores"